src/components/ErrorHandler/ErrorHandler.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert, { AlertProps } from '@mui/material/Alert';
+import { useDispatch } from 'react-redux';
+
+import { useAppSelector } from '../../store/store';
+import { changeError } from '../../store/reducers/errorHandler-reducer';
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>((props, ref) => (
+  <MuiAlert elevation={6} ref={ref} variant='filled' {...props} />
+));
+
+export function ErrorSnackbar() {
+  const noticeText = useAppSelector((state) => state.error.noticeText);
+  const noticeStatus = useAppSelector((state) => state.error.noticeStatus);
+
+  const dispatch = useDispatch();
+  const handleClose = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string
```

Comment:
looks like it should be enum type

Reply:
I imported SnackbarCloseReason type instead of string from Snackbar.d.ts to my component